website/server/models/group.js

```diff
@@ -520,6 +523,9 @@ schema.methods.startQuest = async function startQuest (user) {
         });
     });
   });
+  this.sendChat(`Your quest, ${quest.text('en')}, has started.`, null, {
```

Comment:
I believe we should make this string translatable. That will allow you to remove the meta data addition as well. Unless you are using the meta data for something else. 

add this to common/locals
`"questStarted": "Your quest, <%= questText %>, has started. <%= participatingMembers %>."`

replace the first param of sent chat with
```
res.t('questStarted', { questText: quest.text('en'), participatingMembers: this.getParticipatingQuestMembers().join(', ')});
```